docs/start/install.mdx

```diff
@@ -137,6 +137,121 @@ Additionally, Qiskit only supports CPython. Running with other Python interprete
   - Windows 32 bit for Python 3.10 or later
 </details>
 
+## Qiskit Versioning
+
+Qiskit version numbers follow [Semantic Versioning](https://semver.org/).
+The version number is comprised of 3 primary components, the major, minor, and
+patch versions. For a version number `X.Y.Z` where `X` is the major version,
+`Y` is the minor version, and `Z` is the patch version.
+
+Breaking API changes are reserved for major version releases. The **minimum**
+period between major version releases is one year. Minor versions will be
+periodically (currently every three months) published for the current major
+version which add new features and bug fixes. For the most recent minor version,
+there will also be new patch versions published as bugs are identified and fixed
+on that release series. A tentative release schedule is included below:
+
+![](/images/start/install/release_schedule.png)
+
+For an up-to-date release schedule you should refer to the Qiskit Github project's
+[milestones list](https://github.com/Qiskit/qiskit/milestones), which will always
+contain the current release plan.
+
+With the release of a new major version, the previous major version is supported
+for at least 6 months; only bug and security fixes will be accepted during this
+time and only patch releases will be published for this major version. A final
+patch version will be published when support is dropped and that release will
+also document the end of support for that major version series. A longer
+support window is needed for the previous major version as this gives downstream
+consumers of Qiskit a chance to migrate not only their code but also their
+users. It's typically not recommended for a downstream library that
+depends on Qiskit to bump its minimum Qiskit version to a new
+major version release immediately because its user base also needs a chance
+to migrate to the new API changes. Having an extended support window
+for the previous major Qiskit version gives downstream projects time to fix
+compatibility with the next major version. Downstream projects can provide
+support for two release series at a time to give their users a migration path.
+
+For the purposes of semantic versioning, the Qiskit public API is considered
+any documented module, class, function, or method that is not marked as private
+(with a `_` prefix). However, there can be explicit exceptions made in the case
+of specific documented APIs. In such cases these APIs will be clearly documented
+as not being considered stable interfaces yet and a user visible warning will be
+actively emitted on any use of these unstable interfaces. Additionally, in some
```

Comment:
Maybe "experimental APIs" need it's own subsection as their exceptionality might be important to set the expectations on breaking changes during stable cycles.

Reply:
TBH, I didn't want to call it out too much as I think it's an anti-pattern we wouldn't want to use widely. I just wanted to make sure we document it as something that we can do if needed. I'm not sure we should make it a specific subsection.